src/main/java/edu/cornell/kfs/rass/batch/AwardTranslationDefinition.java

```diff
@@ -84,34 +87,41 @@ public void processCustomTranslationForBusinessObjectCreate(
     }
 
     protected AwardAccount createDefaultAwardAccount(RassXmlAwardEntry xmlAward) {
-        Pair<String, String> defaultChartAndAccount = getChartAndAccountForDefaultAwardAccount();
-        String directorPrincipalId = parameterService.getParameterValueAsString(
-                RassStep.class, RassParameterConstants.DEFAULT_PROJECT_DIRECTOR);
+        Map<String, String> defaultAwardAccountAttributes = getDefaultAwardAccountAttributes();
+        String directorPrincipalId = parameterService.getParameterValueAsString(RassStep.class, RassParameterConstants.DEFAULT_PROJECT_DIRECTOR);
         if (StringUtils.isBlank(directorPrincipalId)) {
             throw new RuntimeException("Default project director parameter cannot be blank");
         }
-        
         AwardAccount awardAccount = new AwardAccount();
         awardAccount.setProposalNumber(xmlAward.getProposalNumber());
         awardAccount.setPrincipalId(directorPrincipalId);
-        awardAccount.setChartOfAccountsCode(defaultChartAndAccount.getLeft());
-        awardAccount.setAccountNumber(defaultChartAndAccount.getRight());
+        awardAccount.setChartOfAccountsCode(defaultAwardAccountAttributes.get(KFSPropertyConstants.CHART));
+        awardAccount.setAccountNumber(defaultAwardAccountAttributes.get(KFSPropertyConstants.ACCOUNT_NUMBER));
+        awardAccount.setActive(defaultAccountIndicatorIsActive(KFSPropertyConstants.ACCOUNT_ACTIVE_INDICATOR));
         return awardAccount;
     }
+    
+    protected boolean defaultAccountIndicatorIsActive(String rowIndicatorValue) {
+        return StringUtils.equals(rowIndicatorValue, KFSConstants.ACTIVE_INDICATOR) ? true : false;
```

Comment:
You can just return the StringUtils.equals() result directly; making it a ternary of "? true : false" is redundant.